packages/protocol/migrations/24_governance.ts

```diff
@@ -95,10 +92,8 @@ module.exports = deploymentForCoreContract<GovernanceInstance>(
       'Validators',
     ]
 
-    await Promise.all(
```

Comment:
does removing this make the tx happen sequentially instead of in parallel? is this related to the ledger flakiness?

Reply:
Correct, now it will be in series, which while slower seems to be more reliable